Utils/request_contributor_review.py

```diff
@@ -65,19 +65,26 @@ def get_pr_modified_files_and_packs(pr_number, github_token, verify_ssl):
     pr_files = [f.get('filename') for f in pr_changed_data]
     modified_packs = {Path(p).parts[1] for p in pr_files if p.startswith(PACKS_FOLDER) and len(Path(p).parts) > 1}
 
-    return modified_packs, pr_files
+    blob_url = pr_changed_data[0].get('blob_url').split('/')
+    commit = blob_url[blob_url.index('blob')+1]
 
+    return modified_packs, pr_files, commit
 
-def tag_user_on_pr(reviewers: set, pr_number: str, pack: str, pack_files: set, github_token: str = None,
+
+def tag_user_on_pr(reviewers: set, pr_number: str, pack: str, pack_files: set, commit: str, github_token: str = None,
                    verify_ssl: bool = True):
     comments_endpoint = f"https://api.github.com/repos/{REPO_OWNER}/{REPO_NAME}/issues/{pr_number}/comments"
     headers = {"Authorization": "Bearer " + github_token} if github_token else {}
 
     reviewers_comment = "\n".join({f"- @{r}" for r in reviewers})
-    pack_files_comment = "\n".join(pack_files)
+    # pack_files_comment = "\n[".join(pack_files) + ']'
+    pack_files_comment = ''
+    new_line = '/n'
+    for file in pack_files:
+        pack_files_comment += f'\n[{file}](https://github.com/demisto/content/blob/{commit}/{file})'
```

Comment:
Any way to display the diff file? (instead of the file committed) like here https://github.com/demisto/content/pull/10649/files#diff-8a7843e1fa2f89499f2bf3354fed238fea235b417bca651fd36a5dbf3b7e1c0e